### DISCUSSION TITLE
Mod Suggestion Replay Mod

### DISCUSSION THREAD

**Discussion by RiskEditz** [Sinytra/Connector#227] [Ideas]
I know Replay mod will be extremely hard to make compatible but I believe doing this would make this project more popular. And with popularity comes support. Doing this would significantly change and improve the forge mod community. Thanks for all your work.

https://www.replaymod.com/docs/

**Comment by mcclellanja** [comment]
I was able to get the Replay mod working just fine. Maybe try with the latest version?

**Comment by coco875** [comment]
> I know Replay mod will be extremely hard to make compatible but I believe doing this would make this project more popular. And with popularity comes support. Doing this would significantly change and improve the forge mod community. Thanks for all your work.
> 
> 
> 
> https://www.replaymod.com/docs/

Replay mod are now compatible with the latest version of connector

**Comment by VladisCrafter** [comment]
It **is** compatible now, since beta 36 which was released a few days ago.

**Reply by Darudas** [reply]
It was and is compatible just not with some other mods together than you could get a crash
